internal/concierge/impersonator/impersonator.go

```diff
@@ -812,6 +813,15 @@ func getTransportForProtocol(restConfig *rest.Config, protocol string) (http.Rou
 		return nil, fmt.Errorf("could not build transport: %w", err)
 	}
 
+	// For clients that support http2, transport.New calls http2.ConfigureTransports,
+	// which configures with both h2 and http/1.1,
+	// even when you explicitly only ask for h2.
+	// Override that change.
+	cfg, err := apimachinerynet.TLSClientConfig(rt)
```

Comment:
This import is already in this file:

```suggestion
	cfg, err := utilnet.TLSClientConfig(rt)
```

Reply:
Whoops! I totally missed that somehow!